lib/LANraragi/Model/Plugins.pm

```diff
@@ -1,5 +1,9 @@
 package LANraragi::Model::Plugins;
 
+use v5.36;
+use experimental 'try';
+no warnings 'experimental::try';
```

Comment:
Does this work for you? My perl 5.36 still yells at me about try/catch being experimental when I added this in the copyarchivetags plugin on my own 🤔

Reply:
I know, it only removed a couple of lines from the log in foreground mode, but it keeps spamming the tests.
So I'm going to put it inside every sub with a "try/catch". I don't like it very much, but it seems the way, at least until you decide to upgrade to v5.40.

Reply:
Right, I guess we'll just have to deal with it then.

I can't really require 5.40 until it lands in debian stable, that'd be annoying for everyone. We'll probably have to wait a few years for that though 😤

Reply:
Maybe no, unless there are specific reasons, you can try the image "perl:5.40.0-bullseye". It's an official image.